### ISSUE TITLE
JSON decode error when trying to play episodes

### ISSUE THREAD

**Issue by Prashanthvsdvn** [sdaqo/anipy-cli#118]
**Describe the bug**
Script fails trying to play episode, citing JSON decode error.

**To Reproduce**
1. Open `anipy-cli`
2. Search and select a series
3. Crash

**Expected behavior**
Script allows us to pick the episode for the selected series for viewing

**Info**
```
Name: anipy-cli
Version: 2.7.9
Summary: Little tool in python to watch anime from the terminal (the better way to watch anime)
Home-page: https://github.com/sdaqo/anipy-cli
Author: sdaqo
Author-email: sdaqo.dev@protonmail.com
License: GPL-3.0
Location: C:\Python311\Lib\site-packages
Requires: beautifulsoup4, better-ffmpeg-progress, m3u8, moviepy, pycryptodomex, pypresence, python-dateutil, python-mpv, pyyaml, requests, setuptools, tqdm
Required-by:
```

**Platform**
 - OS: Windows 11

**Additional context**
Trace back from script
```
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\requests\models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\json\decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 4 (char 3)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Python311\Scripts\anipy-cli.exe\__main__.py", line 7, in <module>
  File "C:\Python311\Lib\site-packages\anipy_cli\run_anipy_cli.py", line 8, in main
    cli.run_cli()
  File "C:\Python311\Lib\site-packages\anipy_cli\cli\cli.py", line 37, in run_cli
    cli_class(options=args, rpc_client=rpc_client).run()
  File "C:\Python311\Lib\site-packages\anipy_cli\cli\clis\base_cli.py", line 36, in run
    self.process()
  File "C:\Python311\Lib\site-packages\anipy_cli\cli\clis\default_cli.py", line 33, in process
    url_parser = videourl(self.entry, self.options.quality)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\anipy_cli\url_handler.py", line 299, in __init__
    keys = self.get_encryption_keys()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\anipy_cli\url_handler.py", line 323, in get_encryption_keys
    for _, __ in self.session.get(self.enc_key_api).json().items()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\requests\models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Extra data: line 1 column 4 (char 3)
```

**Comment by sdaqo** [comment]
will look into it ---

**Comment by sdaqo** [comment]
Ok fixed in 2.7.10 :)

**Comment by sdaqo** [comment]
Please reopen if something is wrong.
